client/src/document/ingredients/polyfills-subsection.tsx

```diff
@@ -0,0 +1,66 @@
+import { DisplayH3 } from "./utils";
+
+const ALLOWED_POLYFILL_SERVICE = [
```

Comment:
Should be plural.

```suggestion
const ALLOWED_POLYFILL_SERVICES = [
```

Reply:
@teoli2003 You marked this as resolved, but the current status of this PR still has singular. This also applies to other threads. Did you push your changes?